### ISSUE TITLE
Could not resolve com.nbsp:library:1.8

### ISSUE THREAD

**Issue by PiPiCaptain** [nbsp-team/MaterialFilePicker#132]
Could not resolve com.nbsp:library:1.8

**Comment by emad3020** [comment]
Yes, I have face the same issue when I've upgraded my Gradle version to 7.0.1 and remove deprecated JCenter(). I think the problem is that the library didn't migrate from the old dependency with JCenter to the new one of MavenCenteral()
